src/main/java/com/agilethought/atsceapi/domain/UserDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.agilethought.atsceapi.domain;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UserDTO {
+	private String id;
+	private Integer type;
+	private String firstName;
+	private String lastName;
+	private String email;
+	private String password;

```

Comment:
additionally, we need to remove sensitive data from UserDTO, we need to remove password from responses of get.